examples/basic/src/pages/index.tsx

```diff
@@ -126,23 +126,51 @@ export default function BasicExample() {
             return (
               <Tr key={key} data-testid="calendar-weeks">
                 {days.map((day) => {
-                  const { key, date, isCurrentDate, isCurrentMonth } = day
+                  const {
+                    key,
+                    date,
+                    isCurrentDate,
+                    isCurrentMonth,
+                    isWeekend,
+                  } = day
+
+                  // FIXME: hard coding
+
+                  if (isWeekend) {
```

Comment:
sadly, `examples/basic` project need to update dependency to next version, **after publishing new version** ... 😢

Reply:
Okay~! I was thinking about checking `undefined`, but I'll update it to another issue later!
I reflected it.